docs/apigateway.md

```diff
@@ -1,53 +1,100 @@
-# API Gateway (Experimental)
+# API Gateway
 
-[Web Actions](webactions.md) are out from experimental phase.
+API Gateway are out from experimental phase.
 
-Web Actions allows you to invoke an action with HTTP methods other than POST and without the action's authorization API key.
+[Web Actions](webactions.md) allows you to invoke an action with HTTP methods other than POST and without the action's authorization API key.
 
 As a result of user feedback, Web Actions are the programming model chosen to build OpenWhisk actions capable of handling HTTP events.
 
-Most of the API Gateway functionality was merged into Web Actions, Web Actions allows you to handle any HTTP request and return HTTP responses with full control from your Web Action.
+The API Gateway can be configured to proxy your Web Actions providing them with API Gateway features such as rate limiting, oauth token validation, API keys, and more.
```

Comment:
How about: "The API Gateway acts as a proxy to Web Actions and  providing them with additional features features including HTTP method routing, rate limiting, oauth token validation, API keys, and more." ?